Solutions/Marain.Workflow.Api.EngineHost.Client/Marain/Workflows/Client/MarainWorkflowEngineOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <copyright file="WorkflowEngineConfiguration.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Marain.Workflows.Client
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+
+    /// <summary>
+    /// A class for storing the configuration for setting up the workflow engine.
```

Comment:
Is it for setting up the workflow engine? Or is this something clients of the workflow engine will use to ensure they are able to reach it? Are we in fact setting up the workflow engine _client_?